### ISSUE TITLE
[FR] is it possible to check which models has embeddings (similarly `model.has_logits`) without downloading all the models?

### ISSUE THREAD

**Issue by massisenergy** [voxel51/fiftyone#5527]
### Proposal Summary

is it possible to check which models has embeddings (similarly model.has_logits) without downloading all the models?

### Motivation

-   What is the use case for this feature?

Get to know about the models, and find out the ones which are useful for praticular purpose, before proceeding to download.

-   Why is this use case valuable to support for FiftyOne users in general?

Otherwise, users have to download everything or know about all the available models beforehand.

-   Why is this use case valuable to support for your project(s) or organization?

-   currently it downloads all the available models, which is impractical.

### What areas of FiftyOne does this feature affect?

-   [✔️] App: FiftyOne application
-   [✔️] Core: Core `fiftyone` Python library
-   [✔️] Server: FiftyOne server

### Details

I used this code to access the `has_embeddings`:

```
import fiftyone.zoo as foz

# Get all available zoo model names
all_models = foz.list_zoo_models()

# List to hold the models that support embeddings
models_with_embeddings = []

# Iterate through each model and check its has_embeddings attribute
for model_name in all_models:
    try:
        # Load the model on GPU 0 (or change device if needed)
        model = foz.load_zoo_model(model_name, device="cuda:0")
        if model.has_embeddings:
            models_with_embeddings.append(model_name)
    except Exception as e:
        # Some models might fail to load, so we catch and skip those
        print(f"Skipping {model_name} due to error: {e}")

print("Models with embeddings support:")
print(models_with_embeddings)

```

### Willingness to contribute

The FiftyOne Community welcomes contributions! Would you or another member of your organization be willing to contribute an implementation of this feature?

-   [ ] Yes. I can contribute this feature independently
-   [ ] Yes. I would be willing to contribute this feature with guidance from the FiftyOne community
-   [✔️] No. I cannot contribute this feature at this time

**Comment by brimoor** [comment]
We're pretty diligent about adding the `embeddings` and `logits` tags to all models that support them, which means you can do this:

```py
import fiftyone.zoo as foz

 print(foz.list_zoo_models(tags="embeddings"))
```

```
['alexnet-imagenet-torch',
 'classification-transformer-torch',
 'clip-vit-base32-torch',
 'densenet121-imagenet-torch',
 'densenet161-imagenet-torch',
 'densenet169-imagenet-torch',
 'densenet201-imagenet-torch',
 'detection-transformer-torch',
 'dinov2-vitb14-reg-torch',
 'dinov2-vitb14-torch',
 'dinov2-vitg14-reg-torch',
 'dinov2-vitg14-torch',
 'dinov2-vitl14-reg-torch',
 'dinov2-vitl14-torch',
 'dinov2-vits14-reg-torch',
 'dinov2-vits14-torch',
 'googlenet-imagenet-torch',
 'inception-resnet-v2-imagenet-tf1',
 'inception-v3-imagenet-torch',
 'inception-v4-imagenet-tf1',
 'mnasnet0.5-imagenet-torch',
 'mnasnet1.0-imagenet-torch',
 'mobilenet-v2-imagenet-tf1',
 'mobilenet-v2-imagenet-torch',
 'open-clip-torch',
 'resnet-v1-50-imagenet-tf1',
 'resnet-v2-50-imagenet-tf1',
 'resnet101-imagenet-torch',
 'resnet152-imagenet-torch',
 'resnet18-imagenet-torch',
 'resnet34-imagenet-torch',
 'resnet50-imagenet-torch',
 'resnext101-32x8d-imagenet-torch',
 'resnext50-32x4d-imagenet-torch',
 'shufflenetv2-0.5x-imagenet-torch',
 'shufflenetv2-1.0x-imagenet-torch',
 'vgg11-bn-imagenet-torch',
 'vgg11-imagenet-torch',
 'vgg13-bn-imagenet-torch',
 'vgg13-imagenet-torch',
 'vgg16-bn-imagenet-torch',
 'vgg16-imagenet-tf1',
 'vgg16-imagenet-torch',
 'vgg19-bn-imagenet-torch',
 'vgg19-imagenet-torch',
 'voxel51/clip-vit-base32-torch',
 'wide-resnet101-2-imagenet-torch',
 'wide-resnet50-2-imagenet-torch',
 'zero-shot-classification-transformer-torch',
 'zero-shot-detection-transformer-torch']
```

and this:

```py
import fiftyone.zoo as foz

print(foz.list_zoo_models(tags="logits"))
```

```
['alexnet-imagenet-torch',
 'classification-transformer-torch',
 'clip-vit-base32-torch',
 'densenet121-imagenet-torch',
 'densenet161-imagenet-torch',
 'densenet169-imagenet-torch',
 'densenet201-imagenet-torch',
 'detection-transformer-torch',
 'googlenet-imagenet-torch',
 'inception-resnet-v2-imagenet-tf1',
 'inception-v3-imagenet-torch',
 'inception-v4-imagenet-tf1',
 'mnasnet0.5-imagenet-torch',
 'mnasnet1.0-imagenet-torch',
 'mobilenet-v2-imagenet-tf1',
 'mobilenet-v2-imagenet-torch',
 'open-clip-torch',
 'resnet-v1-50-imagenet-tf1',
 'resnet-v2-50-imagenet-tf1',
 'resnet101-imagenet-torch',
 'resnet152-imagenet-torch',
 'resnet18-imagenet-torch',
 'resnet34-imagenet-torch',
 'resnet50-imagenet-torch',
 'resnext101-32x8d-imagenet-torch',
 'resnext50-32x4d-imagenet-torch',
 'shufflenetv2-0.5x-imagenet-torch',
 'shufflenetv2-1.0x-imagenet-torch',
 'vgg11-bn-imagenet-torch',
 'vgg11-imagenet-torch',
 'vgg13-bn-imagenet-torch',
 'vgg13-imagenet-torch',
 'vgg16-bn-imagenet-torch',
 'vgg16-imagenet-tf1',
 'vgg16-imagenet-torch',
 'vgg19-bn-imagenet-torch',
 'vgg19-imagenet-torch',
 'voxel51/clip-vit-base32-torch',
 'wide-resnet101-2-imagenet-torch',
 'wide-resnet50-2-imagenet-torch',
 'zero-shot-classification-transformer-torch',
 'zero-shot-detection-transformer-torch']
```

**Comment by brimoor** [comment]
You can also see this via the CLI like so:

```
fiftyone zoo models list --tags embeddings
fiftyone zoo models list --tags logits
```
